sonarpedia.json

```diff
@@ -0,0 +1,7 @@
+{
+  "rules-metadata-path": "./java-checks/src/main/resources/org/sonar/l10n/java/rules/squid",
+  "languages": [
+    "JAVA"
```

Comment:
I am surprised by the case of the language in this file cc @erichirlemann

Reply:
I mean that we have nowhere the java language used in uppercase. The langage key is "java" as defined per https://github.com/SonarSource/sonar-java/blob/master/sonar-java-plugin/src/main/java/org/sonar/plugins/java/Java.java#L39

Reply:
This key comes from rule-api instead.